app/controllers/email_subscriptions_controller.rb

```diff
@@ -15,7 +15,7 @@ def unsubscribe
     if verified_params[:expires_at] > Time.current
       user = User.find(verified_params[:user_id])
       user.update(verified_params[:email_type] => false)
-      @email_type = PREFERRED_EMAIL_NAME[verified_params[:email_type]]
+      @email_type = PREFERRED_EMAIL_NAME.fetch(verified_params[:email_type], "this list")
```

Comment:
Unrelated change?

If this change is intended and works I think it's fine to keep in.

Reply:
Yep intentional, suggested by https://github.com/thepracticaldev/dev.to/pull/6027#discussion_r378629230

Reply:
I see @benhalpern's point, it's confusing to have in this PR as it's not related per se, but I think we can leave it in.